build.gradle

```diff
@@ -39,25 +30,32 @@ repositories {
     maven { url "https://maven.google.com" }
 }
 
+apply plugin: "groovy"
```

Comment:
I've never really thought about it before; what's the benefit of moving plugins before or after `buildscript`?